docs/OtherUseCases.md

```diff
@@ -0,0 +1,56 @@
+# Other Use Cases
+
+## Launching as an Agent
+
+If you need to profile some code as soon as the JVM starts up, instead of using the `asprof`,
+it is possible to attach async-profiler as an agent on the command line. For example:
+
+```
+$ java -agentpath:/path/to/libasyncProfiler.so=start,event=cpu,file=profile.html ...
+```
+
+Agent library is configured through the JVMTI argument interface.
+The format of the arguments string is described
+[in the source code](https://github.com/async-profiler/async-profiler/blob/v3.0/src/arguments.cpp#L44).
+`asprof` actually converts command line arguments to that format.
+
+Another important use of attaching async-profiler as an agent is for continuous profiling.
+
+## Using Java API
+async-profiler Java API is published to maven central. Like any other dependency, we have to
+just include the below dependency.
+```
+<!-- https://mvnrepository.com/artifact/tools.profiler/async-profiler -->
+<dependency>
+    <groupId>tools.profiler</groupId>
+    <artifactId>async-profiler</artifactId>
+    <version>3.0</version>
```

Comment:
This will require updating version with each new release.
Can we avoid updating numbers in the doc with every version increment?

Reply:
Replaced entire block with latest release maven central link which will always redirect to the latest link